ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.18__cnv-odf-ocp4.18-lp-interop.yaml

```diff
@@ -80,8 +78,53 @@ tests:
     - ref: interop-tests-cnv-tests-e2e-deploy
     - ref: interop-tests-openshift-virtualization-tests
     workflow: firewatch-ipi-aws
+  timeout: 6h0m0s
+- as: cnv-odf-tests-aws-ipi-ocp418-fips
+  cron: 0 6 11 2 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      COMPUTE_NODE_TYPE: c5n.metal
+      FIPS_ENABLED: "true"
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+            [
+              {"step": "cucushift-*"           , "failure_type": "all"          , "classification": "Cluster Health Check"                  , "group": { "name" : "cluster" , "priority" : 2}, "jira_additional_labels": ["!default"]                                           },
+              {"step": "*deploy-odf"           , "failure_type": "all"          , "classification": "Operator Setup"                        , "group": { "name" : "lp-tests", "priority" : 1}, "jira_additional_labels": ["!default", "interop-tests"]                          },
+              {"step": "*ocs-tests"            , "failure_type": "pod_failure"  , "classification": "ODF Storage Tests Execution"           , "group": { "name" : "lp-tests", "priority" : 1}, "jira_additional_labels": ["!default", "interop-tests"]                          },
+              {"step": "*ocs-tests"            , "failure_type": "test_failure" , "classification": "ODF Storage Tests Failure"             , "group": { "name" : "lp-tests", "priority" : 1}, "jira_additional_labels": ["!default", "interop-tests"], "jira_project": "RHSTOR"},
+              {"step": "*cnv-tests-e2e-deploy" , "failure_type": "pod_failure"  , "classification": "CNV Tests Execution"                   , "group": { "name" : "lp-tests", "priority" : 2}, "jira_additional_labels": ["!default", "interop-tests"]                          },
+              {"step": "*cnv-tests-e2e-deploy" , "failure_type": "test_failure" , "classification": "CNV Tests Failure"                     , "group": { "name" : "lp-tests", "priority" : 2}, "jira_additional_labels": ["!default", "interop-tests"], "jira_project": "CNV"   }
+            ],
+          "success_rules":
+            [
+              {"jira_project": "CSPIT", "jira_additional_labels": ["!default"]}
+            ]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.18-lp", "self-managed-lp", "cnv-lp",
+        "fips"]'
+      FIREWATCH_DEFAULT_JIRA_ASSIGNEE: lbednar@redhat.com
```

Comment:
Same

(I think that for LP job, someone from LP should be the default contact. Unless it's CNV specific)

Reply:
[see conversation in Slack thread](https://redhat-internal.slack.com/archives/C04QDE5TK1C/p1732547000512159?thread_ts=1732307276.267649&cid=C04QDE5TK1C)